2024-04-09 17:37:42,892 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-09 17:37:42,893 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Configure stats pid to 7982
2024-04-09 17:37:42,894 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /home/russellsb/.config/wandb/settings
2024-04-09 17:37:42,894 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/settings
2024-04-09 17:37:42,895 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-09 17:37:42,896 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-09 17:37:42,897 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:37:42,898 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-09 17:37:42,899 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:37:42,900 INFO    MainThread:7982 [wandb_init.py:_log_setup():521] Logging user logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_173742-7om2lgwn/logs/debug.log
2024-04-09 17:37:42,901 INFO    MainThread:7982 [wandb_init.py:_log_setup():522] Logging internal logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_173742-7om2lgwn/logs/debug-internal.log
2024-04-09 17:37:42,902 INFO    MainThread:7982 [wandb_init.py:init():561] calling init triggers
2024-04-09 17:37:42,903 INFO    MainThread:7982 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-09 17:37:42,904 INFO    MainThread:7982 [wandb_init.py:init():586] re-initializing run, found existing run on stack: uo18n3ud
2024-04-09 17:37:42,906 INFO    MainThread:7982 [wandb_run.py:_finish():2064] finishing run russellsb/x-ray-chest-disease-classification/uo18n3ud
2024-04-09 17:37:42,908 ERROR   MainThread:7982 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-09 17:37:42,909 INFO    MainThread:7982 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-09 17:37:42,909 INFO    MainThread:7982 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-09 17:37:42,910 INFO    MainThread:7982 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-09 17:37:42,911 INFO    MainThread:7982 [wandb_run.py:_restore():2287] restore
2024-04-09 17:37:42,912 INFO    MainThread:7982 [wandb_run.py:_restore():2293] restore done
2024-04-09 17:37:47,648 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-09 17:37:47,649 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-09 17:37:47,660 INFO    MainThread:7982 [wandb_run.py:_footer_sync_info():3895] logging synced files
2024-04-09 17:37:47,810 INFO    MainThread:7982 [wandb_init.py:init():611] starting backend
2024-04-09 17:37:47,810 INFO    MainThread:7982 [wandb_init.py:init():615] setting up manager
2024-04-09 17:37:47,816 INFO    MainThread:7982 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-09 17:37:47,817 INFO    MainThread:7982 [wandb_init.py:init():623] backend started and connected
2024-04-09 17:37:47,827 INFO    MainThread:7982 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-09 17:37:47,828 INFO    MainThread:7982 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-09 17:37:47,828 INFO    MainThread:7982 [wandb_init.py:init():715] updated telemetry
2024-04-09 17:37:47,840 INFO    MainThread:7982 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-09 17:37:49,207 INFO    MainThread:7982 [wandb_run.py:_on_init():2357] communicating current version
2024-04-09 17:37:49,327 INFO    MainThread:7982 [wandb_run.py:_on_init():2366] got version response 
2024-04-09 17:37:49,327 INFO    MainThread:7982 [wandb_init.py:init():799] starting run threads in backend
2024-04-09 17:37:49,489 INFO    MainThread:7982 [wandb_run.py:_console_start():2335] atexit reg
2024-04-09 17:37:49,489 INFO    MainThread:7982 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-09 17:37:49,490 INFO    MainThread:7982 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-09 17:37:49,490 INFO    MainThread:7982 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-09 17:37:49,491 INFO    MainThread:7982 [wandb_init.py:init():842] run started, returning control to user process
2024-04-09 17:43:15,191 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-09 17:43:15,191 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Configure stats pid to 7982
2024-04-09 17:43:15,192 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /home/russellsb/.config/wandb/settings
2024-04-09 17:43:15,192 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/settings
2024-04-09 17:43:15,193 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-09 17:43:15,193 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-09 17:43:15,194 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:43:15,194 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-09 17:43:15,194 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:43:15,195 INFO    MainThread:7982 [wandb_init.py:_log_setup():521] Logging user logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_174315-5ukx788b/logs/debug.log
2024-04-09 17:43:15,196 INFO    MainThread:7982 [wandb_init.py:_log_setup():522] Logging internal logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_174315-5ukx788b/logs/debug-internal.log
2024-04-09 17:43:15,196 INFO    MainThread:7982 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f29206c10f0>
2024-04-09 17:43:15,197 INFO    MainThread:7982 [wandb_init.py:init():561] calling init triggers
2024-04-09 17:43:15,198 INFO    MainThread:7982 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-09 17:43:15,198 INFO    MainThread:7982 [wandb_init.py:init():586] re-initializing run, found existing run on stack: 7om2lgwn
2024-04-09 17:43:15,203 INFO    MainThread:7982 [wandb_run.py:_finish():2064] finishing run russellsb/x-ray-chest-disease-classification/7om2lgwn
2024-04-09 17:43:15,204 INFO    MainThread:7982 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-09 17:43:15,206 INFO    MainThread:7982 [wandb_run.py:_restore():2287] restore
2024-04-09 17:43:15,207 INFO    MainThread:7982 [wandb_run.py:_restore():2293] restore done
2024-04-09 17:43:19,895 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-09 17:43:19,896 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-09 17:43:19,911 INFO    MainThread:7982 [wandb_run.py:_footer_sync_info():3895] logging synced files
